custom-metrics-stackdriver-adapter/examples/cross-project-metrics/modified-adapter.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: extensions/v1beta1
```

Comment:
It would be useful to include a comment at the top to explain that it is modified by mounting service account credentials.